packages/rum/README.md

```diff
@@ -361,7 +361,7 @@ A service is an independent, deployable code repository that maps to a set of pa
 [11]: https://www.datadoghq.com/pricing/?product=real-user-monitoring--session-replay#real-user-monitoring--session-replay
 [12]: https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces?tab=browserrum
 [13]: https://docs.datadoghq.com/real_user_monitoring/session_replay/privacy_options?tab=maskuserinput
-[14]: https://docs.datadoghq.com/agent/basic_agent_usage#datadog-site
+[14]: https://docs.datadoghq.com/getting_started/site/
```

Comment:
The new URL is nicer but it does not specify the value of the param to put in the configuration though 😕
Should we update this doc to list which parameter to use for each DC?
something like:
```
SITE | SITE URL | SITE PARAMETER | LOCATION
US1 | https://app.datadoghq.com | datadoghq.com | US
US3 | https://us3.datadoghq.com | us3.datadoghq.com | US
```

Reply:
Doc PR was merged (not deployed yet) and updated this PR accordingly